src/process.py

```diff
@@ -1,42 +1,31 @@
-import yaml
 import task
 from execute import execute_commands
```

Comment:
These two lines import local packages. Do they need to be move downward?

Reply:
Yeah, I neglect this import.